javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/instrumentation/InstrumentationModule.java

```diff
@@ -150,7 +176,7 @@ public boolean isHelperClass(String className) {
    * <p><b>This method is generated automatically</b>: if you override it, the muzzle compile plugin
    * will not generate a new implementation, it will leave the existing one.
    */
-  protected String[] getMuzzleHelperClassNames() {
+  public String[] getMuzzleHelperClassNames() {
```

Comment:
should we return `List<String>`, since it looks like we always wrap the return value in `asList` anyways? probably same too for `getMuzzleReferences` above (not suggesting either for this PR)

Reply:
Yes, that can be easily done for `getMuzzleHelperClassNames`; it'll be a bit more complicated (but certainly doable) for `getMuzzleReferences` too. That could be done together with https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/2925#discussion_r628691545 in one of the next PRs